android-kotlin/common/src/main/java/com/google/android/googlevideodiscovery/common/models/ContinueWatchingEntity.kt

```diff
@@ -9,7 +9,33 @@ data class ContinueWatchingEntity(
     val lastEngagementTimeMillis: Long,
     val continueWatchingType: ContinueWatchingType,
     val profileId: String,
-)
+) {
+    val releaseYear: Int
+        get() = when (entity) {
+            is MovieEntity -> entity.releaseYear
+            is TvEpisodeEntity -> entity.releaseYear
+            is VideoClipEntity -> entity.releaseYear
+            else -> throw IllegalStateException("Unsupported video type for Continue Watching: $entity")
+        }
+    val duration: Duration
+        get() = when (entity) {
+            is MovieEntity -> entity.duration
+            is TvEpisodeEntity -> entity.duration
+            is VideoClipEntity -> entity.duration
+            else -> throw IllegalStateException("Unsupported video type for Continue Watching: $entity")
+        }
+
+    val genre: String
+        get() = when (entity) {
+            is MovieEntity -> entity.genre
+            is TvEpisodeEntity -> entity.genre
+            is VideoClipEntity -> entity.genre
+            else -> throw IllegalStateException("Unsupported video type for Continue Watching: $entity")
```

Comment:
Let's change the `else` to `is TvShowEntity` instead. 

The benefit is that when we add a new entity type in future, we will be warned by compiler that we need to update this place. If we use the generic `else`, we might forget to update some place and the app will crash at runtime.